types/rollup-plugin-css-only/index.d.ts

```diff
@@ -0,0 +1,27 @@
+// Type definitions for rollup-plugin-css-only 3.1
+// Project: https://github.com/thgh/rollup-plugin-css-only
+// Definitions by: Mateusz Szewc <https://github.com/SitamMatt>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+import { Plugin, OutputBundle } from 'rollup';
+
+declare namespace css {
+    interface Options {
+        /**
+         *  All CSS files will be parsed by default, but you can also specifically include/exclude files
+         */
+        include?: ReadonlyArray<string | RegExp> | string | RegExp | null;
+        exclude?: ReadonlyArray<string | RegExp> | string | RegExp | null;
```

Comment:
These are undocumented; sure you want to include them?

Reply:
I've copppied these include/exclude properties from another rollup plugin here, but after your comment I think it's better to fix this